### PR TITLE
Rebranding

### PR DESCRIPTION
**Pull request by mcupak**

### CODE REVIEW

File: SEARCHSPEC.md [hunk 1]
```diff
@@ -918,11 +918,11 @@ The difference between the two formats is the way in which the phenopacket json
 
 # Appendix A: SQL Grammar
 
-ANTLR grammar for GA4GH Search (based on PrestoSQL v. 323, ASL 2.0 license), with the DML and DDL parts removed:
+ANTLR grammar for Data Connect (based on PrestoSQL v. 323, ASL 2.0 license), with the DML and DDL parts removed:
 
 
 ```
-grammar Ga4ghSearch;
+grammar Ga4ghData Connect;
```
Comment (jfuerth) [inline SEARCHSPEC.md]: Should be `grammar DataConnect` (I believe the space is a syntax error)

**Review by jfuerth — APPROVED** [review]
Looks great overall. Just one search/replace woopsie that I noticed.

Nice work streamlining the readme.

File: SEARCHSPEC.md (hunk 1 above)
Comment (mcupak) [reply SEARCHSPEC.md]: Good catch! I intended "Search", as in the search endpoint of Data Connect, but let's go with Data Connect.

File: USECASES.md [hunk 2]
```diff
@@ -1,117 +1,117 @@
...
-# Open Questions
-If you have an answer to one of these questions, please either file an issue,
-open a PR against this document or reach out to the list at
-ga4gh-discovery-search@ga4gh.org.
-
-* How exactly are fuzzy matches for phenotypes being performed today? For
-  example: this term and up the tree, this term and down the tree, other well
-  defined functions or something completely custom?
+﻿# Introduction
+This document attempts to summarize Data Connect use cases collected from key driver
```
Comment (kemp-verily) [inline USECASES.md]: Maybe just drop `Data Connect` and leave `summarize use cases`.  I wasn't using `search` originally as the name of the specification, but a classification of user behaviour.  Alternatively, I would say something like `summarize a variety of use cases`...

**Review by kemp-verily — APPROVED** [review]
Thanks for this, Miro.  One suggestion in the USECASES.md file.

File: USECASES.md (hunk 2 above)
Comment (mcupak) [reply USECASES.md]: Done, thank you!

**Comment by mcupak** [comment]
Received feedback from the secretariat to incorporate here: https://docs.google.com/document/d/1lS3kJ_lzJrn0tMSza26Lp3U4L4gX4G0SfHLgB12DuYo/edit#

**Pull request merged**